aws/rust-runtime/aws-runtime/src/service_clock_skew.rs

```diff
@@ -38,11 +35,13 @@ impl From<ServiceClockSkew> for Duration {
     }
 }
 
+/// Interceptor that determines the clock skew between the client and service.
 #[derive(Debug, Default)]
 #[non_exhaustive]
-pub struct ServiceClockSkewInterceptor {}
+pub struct ServiceClockSkewInterceptor;
```

Comment:
Now that #2878 has been merged, this interceptor can have a name `ServiceClockSkewInterceptor`.